base/math.jl

```diff
@@ -716,7 +716,10 @@ end
 ## rem2pi-related calculations ##
 
 function add22condh(xh::Float64, xl::Float64, yh::Float64, yl::Float64)
-    # as above, but only compute and return high double
+    # This algorithm, due to Dekker, computes the sum of
+    # two double-double numbers and return high double.  References:
+    # [1] http://gdz.sub.uni-goettingen.de/dms/load/img/?PPN=PPN362160546_0018&DMDID=DMDLOG_0023&LOGID=LOG_0023&PHYSID=PHYS_0232
```

Comment:
This seems like an overly long and likely-to-be-transient URL. Anything shorter/more permanent?